protected/humhub/modules/space/widgets/views/myMembership.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use humhub\widgets\PanelMenu;
+
+/**
+ * @var string $role
+ * @var array $permissions
+ * @var string $memberSince
+ */
+?>
+
+<div class="panel panel-default panel-my-membership" id="my-membership-panel">
+    <?= PanelMenu::widget(['id' => 'space-my-membership-panel']); ?>
+    <div class="panel-heading"><?= Yii::t('SpaceModule.base', '<strong>About</strong> my membership'); ?></div>
```

Comment:
Could also do the same here as "About my membership is rather generic and you want the user to know it's truly their membership and just calling it "About my membership"  seems lackluster

Reference: https://github.com/humhub/humhub/pull/4442/files#r501692340